packages/react/src/components/pickers/TagPicker/TagItem.styles.ts

```diff
@@ -71,7 +65,14 @@ export function getStyles(props: ITagItemStyleProps): ITagItemStyles {
           },
         },
       },
-      selected && !disabled && [classNames.isSelected],
+      selected &&
+        !disabled && [
+          classNames.isSelected,
+          {
+            background: palette.themePrimary,
+            color: palette.white,
+          },
+        ],
```

Comment:
Are you supposed to be able to select all the items with ctrl+a when not focusing on one item? Currently you can only select all the items if you are focused on one. Other than that, lgtm.

Reply:
yeah, that was the old behavior and is the current behavior. I agree it'd be nice to support both in the future.

Reply:
probably a limitation of how we are using SelectionZone. If focus isn't on a selectable item, ctrl+a doesn't do anything.